style.css

```diff
@@ -0,0 +1,76 @@
+@import url("https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;700&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-color: rgb(244, 244, 244);
+  font-family: "Josefin Sans", sans-serif;
+
+}
+a{
+  text-decoration: none;
+  color: black;
+  font-size: 18px;
+}
+#navbar{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  height: 50px;
+  align-items: center;
+  background-color: rgb(244, 243, 243);
+  position: sticky;
+}
+#welcome-section{
+  padding: 30px;
+  margin-top: 1rem;
+}
+#welcome-section h1 {
+  font-size: 32px;
+  font-family: "Josefin Sans", sans-serif;
+  margin-left: 100px;
+}
+
+#welcome-section {
+  background-color: rgb(189, 189, 190);
+  padding: 30px;
+  width: 100%;
+  height: 5em;
+}
+#projects h2{
+  font-size: 24px;
+  margin-left: 100px;
+  padding: 0px 400px 400px 0px;
+}
+#projects a{
+  font-size: 18px;
+  margin-left: 100px;
+  border-bottom: 1px solid blue;
+  color: blue;
+  padding: 0px;
+}
+#profile-link{
+  padding: 50px;
+  margin-left: 25%;
+}
+#profile-link button{
+  padding: 15px;
+  margin-left: 25%;
+  border: none;
+  background-color: rgb(164, 163, 163);
+  border-radius: 10px;
+  cursor: pointer;
+  color: blue;
+  font-family: "Josefin Sans", sans-serif;
+  font-size: 20px;
+}
+img{
+  height: 100%;
+  object-fit: cover;
+}
+@media (min-width: 480px) and (max-width:1200px){
+  body{
+    background-color: rgb(207, 203, 203);
+}
+}
```

Comment:
Be kind and follow `coding best practices` while coding to have clean and very standardized codes.